slices/libx264-164.yaml

```
@@ -0,0 +1,15 @@
+package: libx264-164
+
+essential:
+  - libx264-164_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libx264.so.164*:
```

```suggestion
      /usr/lib/*-linux-*/libx264.so.164:
```
Wildcard not needed since there is only one such file.

---

Thanks for the review, updated the PR accordingly and  signed the CLA